packages/emotion/README.md

```diff
@@ -47,65 +33,324 @@ const myStyle = css`
-- [emotion-vue](https://github.com/egoist/emotion-vue)
-- [ember-emotion](https://github.com/alexlafroscia/ember-emotion)
-- [CSS to emotion transform](https://transform.now.sh/css-to-emotion/)
-- [ShevyJS](https://github.com/kyleshevlin/shevyjs)
-- [design-system-utils](https://github.com/mrmartineau/design-system-utils) - Utilities to give better access to your design system.
+  70% {
+    transform: translate3d(0, -15px, 0);
+  }
 
-### In the Wild
+  90% {
+    transform: translate3d(0,-4px,0);
+  }
+`
+
+
+const Avatar = () => {
+  return (
+    <div
+      className={css`
+        width: 96px;
+        height: 96px;
+        border-radius: 50%;
+        animation: ${bounce} 1s ease infinite;
+        transform-origin: center bottom;
+      `}
+    >
+  )
+}
+
+render(<Avatar src={logoUrl} />)
+```
 
-- [healthline.com](https://www.healthline.com)
-- [nytimes.com](https://www.nytimes.com)
-- [vault.crucible.gg](http://vault.crucible.gg/)
-- [saldotuc.com](https://saldotuc.com)
-- [gatsbythemes.com](https://gatsbythemes.com/)
-- [blazity.com](https://blazity.com/)
-- [postmates.com](https://postmates.com/)
-- [thedisconnect.co](https://thedisconnect.co/one)
-- [zefenify.com](https://zefenify.com/about.html)
-- [sentry.io](https://sentry.io)
+**Object Styles**
 
-## Contributors
+```jsx
+// @live
+import { css, keyframes } from 'emotion'
+
+const bounce = keyframes({
+  'from, 20%, 53%, 80%, to': {
+    transform: 'translate3d(0,0,0)'
+  },
+  '40%, 43%': {
+    transform: 'translate3d(0, -30px, 0)'
+  },
+  '70%': {
+    transform: 'translate3d(0, -15px, 0)'
+  },
+  '90%': {
+    transform: 'translate3d(0, -4px, 0)'
+  }
+})
+
+
+const Avatar = () => {
+  return (
+    <div
+      className={css({
+        width: 96,
+        height: 96,
+        borderRadius: '50%',
+        animation: `${bounce} 1s ease infinite`,
+        transformOrigin: 'center bottom'
+      })
+    >
+  )
+}
+
+render(<Avatar src={logoUrl} />)
+```
 
-This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
-<a href="graphs/contributors"><img src="https://opencollective.com/emotion/contributors.svg?width=890&button=false" /></a>
+## cx
 
-## Backers
+`cx` is emotion's version of the popular [`classnames` library](https://github.com/JedWatson/classnames).
+`cx` is useful when combining multiple class names, even class names from your stylesheets and emotion generated class names.
 
-Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/emotion#backer)]
+**Key Features**
 
-<a href="https://opencollective.com/emotion#backers" target="_blank"><img src="https://opencollective.com/emotion/backers.svg?width=890"></a>
+- High performance integration with emotion
+- Custom class names. e.g, `.my-bem--class`, are appended in order.
+- Combines the actual content of emotion generated class names. Multiple emotion generated class names are input and a unique class name is output.
```

Comment:
I don't care that `Multiple emotion generated class names are input and a unique class name is output.` I care that it solves all my composition problems, say that and why it does that.